src/locales/en/translation.json

```diff
@@ -122,8 +122,8 @@
   },
   "pages": {
     "manageYourAccount": {
-      "title": "Your GOV.UK account",
```

Comment:
can you rebase onto latest master?
seems to be adding a lot of changes to the PR that shouldn't be here